urumanifest/constants.py

```diff
@@ -81,6 +83,8 @@ class _directorytuple(NamedTuple):
     "external64": _directorytuple("", "client/windows_amd64/external"),
     "internal": _directorytuple("", "client/windows_ia32/internal"),
     "internal64": _directorytuple("", "client/windows_amd64/internal"),
+    "macexternal": _directorytuple("", "client/mac/external"),
+    "macinternal": _directorytuple("", "client/mac/internal"),
```

Comment:
The filename case doesn't match above. Also, just to make sure - we are assuming that mac clients are universal binaries?

Reply:
Yes, thats the assumption. It wouldn't really make sense to have it be a universal binary locally and then a processor specific binary.

That said - that will make things messy when Apple eventually gets around to cutting Intel support from their tools. But that eventuality could be dealt with like any other platform we discontinue support for.

Reply:
Note that currently CI is not generating universal binaries (because of vcpkg limitations). In theory we could figure out a way to `lipo` them together into a universal binary when we attach them to the published release.

Reply:
Note to self: the `gather_lut` keys are expected to be all lowercase.